tests/devices/test_default_qubit_tf.py

```diff
@@ -313,7 +313,7 @@ def test_invalid_state_prep(self):
         """Test that an exception is raised if a state preparation is not the
         first operation in the circuit."""
         dev = DefaultQubitTF(wires=2)
-        state = np.array([0, 12])
+        state = np.array([0, 1])

```

Comment:
The error: 

```python
        with pytest.raises(
            qml.DeviceError,
            match=r"cannot be used after other Operations have already been applied",
        ):
            dev.apply([qml.PauliZ(0), qml.QubitStateVector(state, wires=[0])])
```
is detected but the error that the state is not normalized is displayed first. That is why I have changed the test. I don't know if this is possible